reset/reset.go

```diff
@@ -13,35 +13,55 @@ type DeleteService interface {
 	DeleteCollection(ctx context.Context, lopts metav1.ListOptions) error
 }
 
+type DeleteWithNamespaceService interface {
+	DeleteCollection(ctx context.Context, namespace string, lopts metav1.ListOptions) error
+}
+
 type SchedulerService interface {
 	ResetScheduler() error
 }
 
 // Service cleans up resources.
 type Service struct {
 	client clientset.Interface
-	// deleteServices has the all services for each resource.
+	// deleteServices has the all services for each non-namespaced resource.
 	// key: service name.
 	deleteServices map[string]DeleteService
-	schedService   SchedulerService
+	// deleteNamespacedServices has the all services for each namespaced resource.
+	// key: service name.
+	deleteWithNamespaceService map[string]DeleteWithNamespaceService
```

Comment:
```suggestion
	deleteServiceForNamespacedResources map[string]DeleteWithNamespaceService
```

Reply:
Please change interface name as well. DeleteWithNamespaceService sounds like it will delete resource and namespace.

Reply:
That's true. Fixed👌